app/uk/gov/hmrc/transitmovementsrouter/services/RoutingService.scala

```diff
@@ -91,15 +107,35 @@ class RoutingServiceImpl @Inject() (messageConnectorProvider: EISConnectorProvid
     payload: Source[ByteString, _]
   )(implicit hc: HeaderCarrier): EitherT[Future, RoutingError, Unit] = {
 
-    val officeOfDeparture       = payload.toMat(officeOfDepartureSink(messageType))(Keep.right).run()
+    val sink: Either[RoutingError, Sink[ByteString, Future[ParseResult[CustomsOffice]]]] = messageType match {
```

Comment:
Can we update MovementType to be a sealed abstract class as we have defined done: https://github.com/hmrc/transit-movements/blob/2441c9e9b2f37b16d96fd7417f2ea1a60386a80e/app/uk/gov/hmrc/transitmovements/models/MovementType.scala

It should only ever equal departure or arrival

Reply:
I agree but we should create a new ticket for this, otherwise we'll get ticket scope creep.

Reply:
Are we actaully using `MovementType` anyway? This is `messageType` here, and I couldn't see any usage of `MovementType`.

Reply:
We don't use messageType for this. The point regarding restricting the values is valid, but may be for another ticket.